<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AstronomerFormClass</class>
 <widget class="QMainWindow" name="AstronomerFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AstronomerForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Add new star</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Right Ascension:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Declination:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Diameter:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QListView" name="listView"/>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBoxStars">
      <property name="text">
       <string>See the stars you want to study</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEditFilter"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonView">
      <property name="text">
       <string>View</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AstronomerForm.qrc"/>
 </resources>
 <connections/>
</ui>
